src/XmlToArrayConverter.php

```diff
@@ -129,4 +129,16 @@ private function getElementChildNodes(DOMElement $domElement): array
 
         return $childNodes;
     }
+
+    /**
+     * @param DOMElement $domElement
+     * @param DOMText $domText
+     * @return string
+     */
+    private function trim(DOMElement $domElement, DOMText $domText): string
+    {
+        return in_array($domElement->tagName, $this->config->getTrimExcept())
```

Comment:
I'm using the tag name here instead of calling [getElementName()](https://github.com/clearstream/xml-to-array/blob/ffe8710ffcc7d3ab7dac0727b3beb157e89583f3/src/XmlToArrayConverter.php#L96) to make it more deterministic so it works the same regardless of the [detachNamespace](https://github.com/clearstream/xml-to-array/blob/ffe8710ffcc7d3ab7dac0727b3beb157e89583f3/src/XmlToArrayConfig.php#L7) setting.

But I'm not 100% sure about this to be honest, maybe we should actually use `getElementName()`? What are your thoughts? I'm torn.